content/info/software.md

```diff
@@ -0,0 +1,21 @@
++++
+title = "Software development"
+date = 2024-01-13
++++
+
+## Contract software development
+
+We offer fullstack contract software development. Main areas of expertise are:
```

Comment:
and then

> We offer fullstack contract software development.

We offer full-stack contract software development and consulting services.

Reply:
```suggestion
We offer full-stack contract software development and consulting services. Main areas of expertise are:
```